core/domain/exp_services.py

```diff
@@ -237,6 +237,22 @@ def get_multiple_explorations_by_id(exp_ids, strict=True):
     return result
 
 
+def get_displayable_exploration_summaries_matching_ids(exploration_ids):
+        """Pass in list of exploration ids,
```

Comment:
This is over-indented: de-indent all this by 4.

Also, suggested rewording: Given a list of exploration ids, filters the list for explorations that are currently non-private and not deleted, and returns a list of the corresponding exploration summaries.